raiden/tasks.py

```diff
@@ -1,17 +1,33 @@
 import time
-import structlog
+import threading
+from pkg_resources import parse_version, require
 
 import gevent
 from gevent.event import AsyncResult
-from gevent.queue import (
-    Queue,
-)
+from gevent.queue import Queue
+import structlog
+
 from raiden.exceptions import RaidenShuttingDown
 
+CHECK_VERSION_INTERVAL = 60 * 5
 REMOVE_CALLBACK = object()
 log = structlog.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+def check_version():
+    """Check every 3 hours for a new release"""
+    app_version = parse_version(require('raiden')[0].version)
+    while True:
+        content = grequests.get('https://api.github.com/repos/raiden-network/raiden/releases/latest').json()
+        # getting the latest release version
+        latest_release = parse_version(content['tag_name'])
+        # comparing it to the user's application
+        if app_version < latest_release:
+            print('\033[94m' + '\n\tYou are running an old version.\nPlease update your client!')
```

Comment:
You can use [`click.secho()`](http://click.pocoo.org/6/utils/#ansi-colors) here (instead of raw ansi) which automatically handles various special cases.

Also it would be nice to include the versions in the message and maybe a link to the releases page.